src/arch/helpersve.h

```diff
@@ -609,3 +647,64 @@ static INLINE vfloat vrev21_vf_vf(vfloat vf) {
 static INLINE vint2 veq_vi2_vi2_vi2(vint2 x, vint2 y) {
   return svsel_s32(svcmpeq_s32(ptrue, x, y), ALL_TRUE_MASK, ALL_FALSE_MASK);
 }
+
+
+// Operations for DFT
+
+static INLINE vdouble vposneg_vd_vd(vdouble d) {
+  vmask pnmask = svreinterpret_s32_u64(svlsl_n_u64_x(ptrue, svindex_u64(0, 1), 63));
```

Comment:
I think that's not a good idea since it includes three (or two) FP add operations, and FP operations are considered to be expensive and slow. Those operations are also dependent of the output by the previous instruction. We assume that the ALUs for the unmasked elements are not used, but that might not be the case since power-gating may take some time to kick in.

Reply:
I came up with a good way to remove vmask without additional FP operation.
This should reduce register pressure.
```
static INLINE vdouble vposneg_vd_vd(vdouble d) {
  return svneg_f64_m(d, svdupq_n_b64(false, true), d);
}
```